src/visitors/dependencies.js

```diff
@@ -86,6 +86,15 @@ module.exports = {
 };
 
 function addDependency(asset, node, opts = {}) {
+  if (asset.options.target !== 'browser') {
+    const isRelativeImport =
+      node.value.startsWith('/') ||
+      node.value.startsWith('./') ||
+      node.value.startsWith('../');
```

Comment:
You could do this with a regex maybe? `/^[./]/.test(node.value)`.

Also, I wonder if this is always the desired behavior. Are there cases where you'd want to bundle node_modules into a single file when targeting node?

Reply:
I believe using startsWith will be faster than using a regex, and it also cleaner to read. If you disagree I can change it still.

The logic is taken from typescript documentation: https://www.typescriptlang.org/docs/handbook/module-resolution.html, see the section about "Relative vs. Non-relative module imports".

I think there are cases where you would want to bundle everything, but it would break a lot of packages that expect to dynamically read a file from itself, or to list other packages like parcel itself does for plugins. Too many will break. Perhaps it can be solved in the future, wrapping some fs calls. But it's much more complicated than this solution. And for who is using node, having the modules there is very ok.

Reply:
OK, this is a good start. Let's see if bundling module sources for node comes up later.

Reply:
>And for who is using node, having the modules there is very ok.

I disagree. Currently I bundle all my Nodejs apps' node_modules so I have a single deployable which makes a redeployment easy and avoid any possibility of transitive module breakages.

However I also understand that this isn't an easy problem for a bundler to solve since there are a lot of edge cases working with modules that interact of with binary files and filesystems.
But it would be great to see some work after this lands to help solve some of these issues.

Reply:
@Toxicable https://github.com/parcel-bundler/parcel/pull/1196

Reply:
Hey I'm looking for a few key features relative to Microbundle. It looks like tilde imports don't work?

It may have changed since this PR but the logic should be checking that it starts with `/`, `~`, `.` which should _not_ be measurably slower with a regex `/^[/~.]/`.

https://github.com/parcel-bundler/parcel/blob/f431b78047bc4fde720be1d7f1411cc19710eda6/src/Resolver.js#L121-L147

That function makes me wonder if the logic should be duplicated though.